kinto/core/resource/schema.py

```diff
@@ -213,11 +215,15 @@ class CollectionQuerySchema(QuerySchema):
     _limit = QueryField(colander.Integer())
```

Comment:
Well limit is bound by the config max limit so it shouldn't be a problem, also I don't know if LIMIT is supposed to be a BIGINT or of another kind. I might worth to have a look.